Dockerfile

```diff
@@ -56,9 +56,10 @@ apt-get purge --auto-remove -y gcc make bison flex git curl xz-utils && \
 # Needed for system minimal requirements checkers
 apt-get install -y --no-install-recommends libgcc-5-dev binutils && \
 apt-get clean && \
-rm -fr /root /var/lib/apt/lists/* /tmp/* /var/tmp/* /usr/local/go && \
-echo '#!/usr/bin/env bash\ncp /opt/cni/bin/cilium-cni /tmp/cni/bin && /usr/bin/cilium-agent $@' > /home/with-cni.sh && \
-chmod +x /home/with-cni.sh
+rm -fr /root /var/lib/apt/lists/* /tmp/* /var/tmp/* /usr/local/go
+
+ADD plugins/cilium-cni/cni-install.sh /cni-install.sh
```

Comment:
You can copy the file from ` /tmp/cilium-net-build/src/github.com/cilium/cilium/plugins/cilium-cni/cni-install.sh` before the `rm -fr` command.